pages/index.tsx

```diff
@@ -22,12 +23,16 @@ import { addSentryBreadcrumb } from '../src/services/ErrorReporter'
 import { BreadcrumbCategory } from '../src/services/ErrorReporter/addSentryBreadcrumb'
 
 const AWELL_BRAND_COLOR = '#004ac2'
+
 const Home: NextPage = () => {
   const { t } = useTranslation()
   const { loading, session, branding, error, refetch } = useHostedSession()
   const { removeItem: removeAccessToken } = useLocalStorage('accessToken', '')
   const router = useRouter()
 
+  const [isCloseConfirmationModalOpen, setIsCloseConfirmationModalOpen] =
+    useState(false)
```

Comment:
```suggestion
    useState<boolean>(false)
```

Reply:
no need for this, it is initialized with false, so boolean is inferred